src/server.tsx

```diff
@@ -58,25 +57,28 @@ app.get('/healthcheck', (req: express.Request, res: express.Response) => {
 interface Epic {
     html: string;
     css: string;
+    testName: string;
+    variantName: string;
 }
 
-const fiveMinutes = 60 * 5;
-const [, fetchDefaultEpicContentCached] = cacheAsync(fetchDefaultEpicContent, fiveMinutes);
+const [, fetchConfiguredEpicTestsCached] = cacheAsync(fetchConfiguredEpicTests, 60);
 
 // Return the HTML and CSS from rendering the Epic to static markup
 const buildEpic = async (
     tracking: EpicTracking,
     localisation: EpicLocalisation,
     targeting: EpicTargeting,
 ): Promise<Epic | null> => {
-    const variant = await fetchDefaultEpicContentCached();
+    const tests = await fetchConfiguredEpicTestsCached();
+    const result = findVariant(tests, targeting);
 
-    // Only log for now - as Frontend does this and we may have bugs
-    // We'll need to do more than log when we start our DCR test though
-    if (shouldNotRenderEpic(targeting)) {
+    if (!result) {
         console.log(`Should not render for targeting data: ${JSON.stringify(targeting)}`);
+        return null;
```

Comment:
This change was also made in #69, but didn't want to depend on/branch off of that PR. We will need to rebase once it's been merged though.